supplier_app/constants/countries.py

```diff
@@ -0,0 +1,14 @@
+from collections import namedtuple
+from django.utils.translation import ugettext_lazy as _
+
+DBTuple = namedtuple('DBTuple', 'value verbose_name')
+
+COUNTRIES = {
+    "ARG": DBTuple("ARG", _("Argentina")),
```

Comment:
Please consider using the standard way to name countries: https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes

Reply:
Done